gyb.py

```diff
@@ -363,6 +367,8 @@ def get(self, method):
     "groupsmigration": QuotaBucket(10, 1, 10),
 }
 
+def getConfigPath():
```

Comment:
Not sure the function is needed here unless we expect more complex logic to find config folder in the future.

Reply:
Yeah, I'm not a python guy so I tried to make it as similar as possible. But went ahead and removed the indirection.